src/services/codefixes/fixAddMissingMember.ts

```diff
@@ -62,15 +79,21 @@ namespace ts.codefix {
 
         const leftExpressionType = skipConstraint(checker.getTypeAtLocation(parent.expression)!);
         const { symbol } = leftExpressionType;
-        const classDeclaration = symbol && symbol.declarations && find(symbol.declarations, isClassLike);
-        if (!classDeclaration) return undefined;
+        if (!symbol || !symbol.declarations) return undefined;
 
-        const makeStatic = (leftExpressionType as TypeReference).target !== checker.getDeclaredTypeOfSymbol(symbol);
-        const classDeclarationSourceFile = classDeclaration.getSourceFile();
-        const inJs = isSourceFileJavaScript(classDeclarationSourceFile);
-        const call = tryCast(parent.parent, isCallExpression);
-
-        return { token, classDeclaration, makeStatic, classDeclarationSourceFile, inJs, call };
+        const classDeclaration = find(symbol.declarations, isClassLike);
+        if (classDeclaration) {
+            const makeStatic = (leftExpressionType as TypeReference).target !== checker.getDeclaredTypeOfSymbol(symbol);
+            const classDeclarationSourceFile = classDeclaration.getSourceFile();
+            const inJs = isSourceFileJavaScript(classDeclarationSourceFile);
+            const call = tryCast(parent.parent, isCallExpression);
+            return { token, declaration: classDeclaration, makeStatic, classDeclarationSourceFile, inJs, call };

```

Comment:
consider adding a discriminant here, e.g. `kind: "enum"` vs `kind: "class"`. this way you can skip the function `isEnumInfo`.